test/io/cacheable.test.ts

```diff
@@ -1,24 +1,26 @@
-import { describe, expect, jest, test } from '@jest/globals';
-import * as Keyv from 'keyv';
+import Keyv from 'keyv';
 import { cacheable } from 'src/io/cacheable';
+import { describe, expect, test, vi } from 'vitest';
 
-jest.mock('keyv', () => {
+vi.mock('keyv', () => {
   /* eslint-disable @typescript-eslint/no-explicit-any */
   const values: { [k: string]: any } = {};
-  const get = jest.fn((key: string): any => values[key]);
-  const set = jest.fn((key: string, value: any) => {
+  const get = vi.fn((key: string): any => values[key]);
+  const set = vi.fn((key: string, value: any) => {
     values[key] = value;
   });
   /* eslint-enable @typescript-eslint/no-explicit-any */
-  return class InMemoryKeyV {
-    static get = get;
-    static set = set;
-    public get = get;
-    public set = set;
+  return {
```

Comment:
mocks work a little differently, mainly that the mock factory returns an object with a default property. Maybe instead of this we could wrap the object in some sort of 'default' helper function.